web/src/components/OrganizationContentInfo.tsx

```diff
@@ -32,7 +33,7 @@ const OrganizationContentInfo = ({ organization }: OrganizationContentInfoProps)
   return (
     <StyledCard elevation={2}>
       <Stack direction={mobile ? 'column' : 'row'} padding={2} gap={2}>
-        <Stack maxWidth={160} height={80} alignItems='flex-start'>
+        <Stack height={80} alignItems='flex-start'>
```

Comment:
❓ is there a reason we used flex-start here?
we can add `alignItems='center'` to the first stack.

```suggestion
<Stack direction={mobile ? 'column' : 'row'} padding={2} gap={2} alignItems='center'>
        <Stack height={80} alignItems='flex-start'>
```

<img width="979" height="135" alt="Image" src="https://github.com/user-attachments/assets/452a09f6-caee-4f11-ad57-3797a5324f1a" />

Reply:
On mobile it looks a lot better imo if its aligned to the start. Also on desktop `alignItems='center'` only looks better if the image has white background/padding on top. If the text is aligned to the top, `flex-start` is better imo. Since it therefore depends on the specific image in question, I'd keep it as it is.